### DISCUSSION TITLE
Could you add a way to interrupt the processing?

### DISCUSSION THREAD

**Discussion by eo407** [thygate/stable-diffusion-webui-depthmap-script#99] [Ideas]
I started a process that is taking too long and I want to cancel it but I don't see a way of doing that except to close my terminal window down.

**Comment by thygate** [comment]
The diffusion process can be interrupted in between steps, but the models used in the extension are one shot. It might be possible for Boost, as the individual steps are processed within the script, I have no idea how implement it tho. I might look into it later, ..
